copy_number/facetsFillGeneCN.R

```diff
@@ -1,5 +1,8 @@
 #!/usr/bin/env Rscript
 # fill facets gene CN file
+
+suppressPackageStartupMessages(library("facets", lib.loc="/home/bermans/R-dev/"))
```

Comment:
this script shouldn't need the facets library at all actually. I think that was just copied over earlier.